src/Config/spotify.example.js

```diff
@@ -4,7 +4,18 @@
  */
 
 const spotifyConfig = {
-  clientId: "???"
+  clientId: "???",
+  authEndpoint: 'https://accounts.spotify.com/authorize',
+  redirectUri: "http://localhost:3000/auth",
+  // https://developer.spotify.com/documentation/general/guides/authorization/scopes/
+  scopes: [
+    "user-read-currently-playing",
+    "user-read-playback-state",
+    "user-modify-playback-state",
+    "playlist-modify-private",
+    "ugc-image-upload",
+  ],
+  clientSecret: "???",
```

Comment:
`clientSecret` shouldn't be added. Because this file gets send to all clients, the secret shouldn't be used.

Reply:
This was only to show for people that would like to fork later what to put into this file, the actual config file in my setting is under /Config/spotify.js. But I can remove it.

Reply:
I know :) But they should be kept in sync to avoid someone (including me & Niko) from putting the wrong things in the config